### DISCUSSION TITLE
Hello world!

### DISCUSSION THREAD

**Discussion by atenajoon** [WeAreMahsaAmini/Core#342] [Introductions]
Hi everyone!
My name is Atena, I am a frontend developer, based in Canada. I can help with coding (React, Unity), design (Figma), and content translation.
We'll make it work!

**Comment by ehsaaanes** [comment]
Salam, ma darim ye app misazim baraye donation, niaz be code nevis darim, lotfan #163 ro check konin, age tamayol be hamkari dashtid tu comment benevisin mamnoon

**Comment by Pink3lephant** [comment]
Hi @atenajoon 

Nice to hear from you. We are working on a project to distribute knowledge on how to create VPN servers that actually work (think of it as meta-Outline, where you can setup several protocols). 

At the moment we are working on our landing page (Nextjs / Strapi], it's in progress but we need more resources to finish it faster. 

Can you please have a look and let me know if you're interested in helping out? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail.
